samples/KvsAudioOnlyStreamingSample.c

```diff
@@ -0,0 +1,264 @@
+
+        // set up audio cpd.
+        pAudioTrack = pStreamInfo->streamCaps.trackInfoList[0].trackId == 1 ? &pStreamInfo->streamCaps.trackInfoList[0]
+                                                                                             : &pStreamInfo->streamCaps.trackInfoList[1];
+        pAudioTrack->codecPrivateData = alawAudioCpd;
+        pAudioTrack->codecPrivateDataSize = KVS_PCM_CPD_SIZE_BYTE;
+        CHK_STATUS(mkvgenGeneratePcmCpd(KVS_PCM_FORMAT_CODE_ALAW, ALAW_AUDIO_TRACK_SAMPLING_RATE, ALAW_AUDIO_TRACK_CHANNEL_CONFIG, pAudioTrack->codecPrivateData,
+                                    pAudioTrack->codecPrivateDataSize));
+    } else {
+        CHK_STATUS(createRealtimeAudioStreamInfoProviderWithCodecs(streamName, DEFAULT_RETENTION_PERIOD, DEFAULT_BUFFER_DURATION, AUDIO_CODEC_ID_AAC, &pStreamInfo));
+
+        // To specify PCM/G.711 use createRealtimeAudioStreamInfoProviderWithCodecs
+        // adjust members of pStreamInfo here if needed
+
+        // set up audio cpd.
+        pAudioTrack = pStreamInfo->streamCaps.trackInfoList[0].trackId == 1 ? &pStreamInfo->streamCaps.trackInfoList[0]
+                                                                                             : &pStreamInfo->streamCaps.trackInfoList[1];
+        pAudioTrack->codecPrivateData = aacAudioCpd;
+        pAudioTrack->codecPrivateDataSize = KVS_AAC_CPD_SIZE_BYTE;
+        CHK_STATUS(mkvgenGenerateAacCpd(AAC_LC, AAC_AUDIO_TRACK_SAMPLING_RATE, AAC_AUDIO_TRACK_CHANNEL_CONFIG, pAudioTrack->codecPrivateData,
+                                    pAudioTrack->codecPrivateDataSize));
+    }
+
+    // use relative time mode. Buffer timestamps start from 0
+    pStreamInfo->streamCaps.absoluteFragmentTimes = FALSE;
+
+    data.startTime = GETTIME();
+    data.firstFrame = TRUE;
+    CHK_STATUS(createDefaultCallbacksProviderWithAwsCredentials(accessKey, secretKey, sessionToken, MAX_UINT64, region, cacertPath, NULL, NULL,
+                                                                &pClientCallbacks));
+
+    if (NULL != getenv(ENABLE_FILE_LOGGING)) {
+        if ((retStatus = addFileLoggerPlatformCallbacksProvider(pClientCallbacks, FILE_LOGGING_BUFFER_SIZE, MAX_NUMBER_OF_LOG_FILES,
+                                                                (PCHAR) FILE_LOGGER_LOG_FILE_DIRECTORY_PATH, TRUE) != STATUS_SUCCESS)) {
+            printf("File logging enable option failed with 0x%08x error code\n", retStatus);
+        }
+    }
+
+    CHK_STATUS(createStreamCallbacks(&pStreamCallbacks));
+    CHK_STATUS(addStreamCallbacks(pClientCallbacks, pStreamCallbacks));
+
+    CHK_STATUS(createKinesisVideoClient(pDeviceInfo, pClientCallbacks, &clientHandle));
+    CHK_STATUS(createKinesisVideoStreamSync(clientHandle, pStreamInfo, &streamHandle));
+
+    data.streamStopTime = streamStopTime;
+    data.streamHandle = streamHandle;
+    data.streamStartTime = defaultGetTime();
+    ATOMIC_STORE_BOOL(&data.firstVideoFramePut, FALSE);
```

Comment:
To better match the context can we change this variable name to `firstAudioFramePut`

Reply:
Removed this variable altogether since it is not being used in the audio only sample program